app/component/home/Hero-2.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Box, Container, Typography } from '@mui/material';
+
+const Hero_Two: React.FC = () => (
+	<Container
+		maxWidth="xl"
+		sx={{
+			backgroundImage:
+				'url(https://img.freepik.com/premium-photo/italian-food-background-with-spaghetti-spices-sauce-view_1220-7077.jpg)',
+			backgroundSize: 'cover',
+			backgroundPosition: 'center',
+			height: { xs: '25vh', md: '50vh', lg: '75vh' },
+			display: 'flex',
+			alignItems: 'center',
+			justifyContent: 'center',
+		}}
+	>
+		<Box
+			sx={{
+				textAlign: 'center',
+				backgroundColor: 'primary.main',
+				width: '700px',
+				borderRadius: '10px',
+				display: 'flex',
+				flexDirection: 'column',
+				alignItems: 'center',
+				justifyContent: 'center',
+				padding: '40px',
+				gap: 2,
+			}}
+		>
+			<Typography
+				variant="h3"
+				sx={{
+					color: '#F8F0E6',
+					fontSize: { xs: '14px', md: '20px', lg: '24px' },
+					fontFamily: 'Montserrat',
+					textAlign: 'center',
+					maxWidth: '566px',
+				}}
+			>
+				Tree nut allergy friendly.
+			</Typography>
+			<Typography
+				variant="h3"
+				sx={{
+					color: '#F8F0E6',
+					fontSize: { xs: '14px', md: '20px', lg: '24px' },
+					fontFamily: 'Montserrat',
+					textAlign: 'center',
+					maxWidth: '566px',
+				}}
+			>
+				No preservatives.
+			</Typography>
+			<Typography
+				variant="h3"
+				sx={{
+					color: '#F8F0E6',
+					fontSize: { xs: '14px', md: '20px', lg: '24px' },
+					fontFamily: 'Montserrat',
```

Comment:
You might want to look into the custom Theme file. Where you can set the font style once. Instead of repeating it over and over. Unless this is just a special case. Check out the mui docs about it. This will be a good way to keep your code D.R.Y.